SimG4Core/MagneticField/src/FieldBuilder.cc

```diff
@@ -15,7 +15,7 @@
 #include "G4FieldManager.hh"
 #include "G4LogicalVolumeStore.hh"
 #include "G4Mag_UsualEqRhs.hh"
-#include "G4TMagFieldEquation.hh"
```

Comment:
better to keep the G4 equation.

Reply:
sure

Reply:
will make all changes once the tests are finished